rosidl_runtime_rs/src/sequence.rs

```diff
@@ -463,14 +511,14 @@ impl<T: SequenceAlloc> Iterator for SequenceIterator<T> {
         self.idx += 1;
         Some(elem)
     }
-
+    /// returns the remaining length of the sequence
     fn size_hint(&self) -> (usize, Option<usize>) {
         let len = (self.seq.size + 1) - self.idx;
         (len, Some(len))
     }
 }
-
 impl<T: SequenceAlloc> ExactSizeIterator for SequenceIterator<T> {
+    /// Returns the length of the sequence as unsigned integer.
```

Comment:
If you want to document this function, then the comment should expand on more than just the function signature.

Checkout the rustdocs for [std::Vec::len](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.len)